src/filetreediff.js

```diff
@@ -73,11 +125,61 @@ export class FileTreeDiffElement extends LitElement {
           />
           Show diff
         </label>
+        ${this.renderArrows()}
       `;
     }
     return nothing;
   }
 
+  previousChunk() {
+    if (!this.chunks.length) {
+      return;
+    }
+    if (this.chunkIndex === 1) {
+      this.chunkIndex = 1;
+    } else if (this.chunkIndex != 1) {
+      this.chunkIndex -= 1;
+    }
+
+    const chunk = this.chunks[this.chunkIndex - 1];
+    this.scrollToChunk(chunk);
+  }
+
+  nextChunk() {
+    if (!this.chunks.length) {
+      return;
+    }
+    if (this.chunkIndex != this.chunks.length) {
+      this.chunkIndex += 1;
+    }
+
+    const chunk = this.chunks[this.chunkIndex - 1];
```

Comment:
Seems this code is the same for previous & next? Shouldn't this be adding a chunk?

```suggestion
    const chunk = this.chunks[this.chunkIndex + 1];
```